regression/global_3dvar.sh

```diff
@@ -0,0 +1,306 @@
+set -x
+
+# Set experiment name and analysis date
+
+exp=$jobname
+
+# Set the JCAP resolution which you want.
+export JCAP=48
+export LEVS=127
+export JCAP_B=$JCAP
+
+# Set runtime directories
+tmpdir=$tmpdir/$tmpregdir/${exp}
+
+# Specify GSI fixed field and data directories.
+fixcrtm=${fixcrtm:-$CRTM_FIX}
+
+
+# Set variables used in script
+UNCOMPRESS=gunzip
+CLEAN=NO
+ncp=/bin/cp
+nln="/bin/ln -fs"
+
+
+# Given the requested resolution, set dependent resolution parameters
+if [[ "$JCAP" = "96" ]]; then
+   export LONA=384
+   export LATA=192
+   export DELTIM=1200
+elif [[ "$JCAP" = "48" ]]; then
+   export LONA=192
+   export LATA=96
+   export DELTIM=1200
+else
+   echo "INVALID JCAP = $JCAP"
+   exit
+fi
+export NLON=$LONA
+export NLAT=$((${LATA}+2))
+
+
+# Given the analysis date, compute the date from which the
+# first guess comes.  Extract cycle and set prefix and suffix
+# for guess and observation data files
+gdate=`date +%Y%m%d%H -d "${global_adate:0:8} ${global_adate:8:2} - 6 hours"`
+PDYa=`echo $global_adate | cut -c1-8`
+cyca=`echo $global_adate | cut -c9-10`
+PDYg=`echo $gdate | cut -c1-8`
+cycg=`echo $gdate | cut -c9-10`
+
+dumpobs=gdas
+prefix_obs=${dumpobs}.t${cyca}z
+prefix_ges=gdas.t${cycg}z
+prefix_ens=gdas.t${cycg}z
+suffix=tm00.bufr_d
+
+dumpges=gdas
+COMROOTgfs=$casesdir/gfs/prod
+datobs=$COMROOTgfs/$dumpobs.$PDYa/${cyca}/atmos
+datges=$COMROOTgfs/$dumpges.$PDYg/${cycg}/atmos
+datens=$COMROOTgfs/enkfgdas.$PDYg/${cycg}/atmos
+
+
+# Set up $tmpdir
+rm -rf $tmpdir
+mkdir -p $tmpdir
+cd $tmpdir
+rm -rf core*
```

Comment:
is this `rm -rf core*` necessary since presumably `$tmpdir` is removed immediately before and recreated?

Reply:
Agreed.   There is no reason to `rm -rf core*`.   Line removed at 1ba3fb54.